### ISSUE TITLE
Wrong starting column when toggling comments for makefile commands

### ISSUE THREAD

**Issue by mcandre** [microsoft/vscode#234464]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.95.2
- OS Version: macOS Sequoia

Steps to Reproduce:

1. Write a makefile with a default task like `all:` and a command like `echo hello`.
2. Use VSCode hotkeys to toggle comments on and off for the command.

Expected:

Code comments for make commands begin at the very first column.

Observed:

Code comments try to indent the comment. This disrupts the visual flow of makefiles, and the syntax risks interacting poorly with parsers for various make implementations and shells.

**Comment by RedCMD** [comment]
```makefile
all:
	echo hello
```
pressing ctrl+/ on line 2 results in the comment being placed after the tab rather than at the start of the line

**Comment by mcandre** [comment]
Yeah, this is a longstanding bug that's been happening for years.

**Comment by Enzo-Nunes** [comment]
@alexr00 @hediet Hello! I'd like to take a look at this issue. Can I be assigned?

**Comment by RedCMD** [comment]
this should be the file https://github.com/microsoft/vscode/blob/main/src/vs/editor/contrib/comment/browser/lineCommentCommand.ts

**Comment by Enzo-Nunes** [comment]
@RedCMD 
Yes, thank you for the assist! I developed a fix and changed a test for the _normalizeInsertionPoint method in src/vs/editor/contrib/comment/test/browser/lineCommentCommand.test.ts


I've read all the coding guidelines.
Should I know anything else before I submit a PR?

**Comment by Enzo-Nunes** [comment]
I see now that this check maybe should be placed elsewhere. I'll keep working on this and will post back here once I've concluded something.
